### DISCUSSION TITLE
BORIS for students with old computer OS's

### DISCUSSION THREAD

**Discussion by bfranks80** [olivierfriard/BORIS#362] [Q&A]
Hello,

I am teaching an animal behavior course and would like to have the students collect data using BORIS.  Several of them are not able to open BORIS on their laptops, however.  We have tried adjusting the "security & privacy" settings in the system preferences, but it hasn't worked in all cases.  It seems to have something to do with having an older OS (eg, 10.15) on Macs.   Is this a known issue? 

One student who was having this problem was able to get around it by installing an older version of BORIS, but I am worried this will create incompatibility issues with the shared project design.  

Any suggestions or directions are appreciated. 

Thank you,

**Comment by olivierfriard** [comment]
Hello,
It should be very useful to known the versions you have tried.
O

On Saturday, February 13, 2021, bfranks80 <notifications@github.com> wrote:
> Hello,
>
> I am teaching an animal behavior course and would like to have the
students collect data using BORIS. Several of them are not able to open
BORIS on their laptops, however. We have tried adjusting the "security &
privacy" settings in the system preferences, but it hasn't worked in all
cases. It seems to have something to do with having an older OS (eg, 10.15)
on Macs. Is this a known issue?
>
> One student who was having this problem was able to get around it by
installing an older version of BORIS, but I am worried this will create
incompatibility issues with the shared project design.
>
> Any suggestions or directions are appreciated.
>
> Thank you,
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.<
https://ci5.googleusercontent.com/proxy/5XcWq_ELzVWdn1g8SYcw_JujQUsGsRmUBJGi44c0u7-Z0E0myJWy6glBQ3uHCojRw5u9Jv-3PURCpdC6VkcU-UunAlkyYBSPKD4rtjXBXjwn9lNU2hTzh0J9ITFzu5WfVb16t835ZGAvtpt6LG0mWW1xIQwQioMYeHt6aqzLpzDmg4hyGFqx61rjcHyWNJUVLX1A7fGWATvYPaNbxSQvN0qYGaKQT5O9Z-7RAw=s0-d-e1-ft#https://github.com/notifications/beacon/ABR2AYR6NPCZ37H6SJIRVILS63TN5A5CNFSM4XSQJ552YY3PNVWWK3TUL52HS4DFVJCGS43DOVZXG2LPN2VGG33NNVSW45C7NFSM4ABRFNBQ.gif
>

-- 
University of Torino
Life Sciences and Systems Biology department
Via dell'Accademia Albertina, 13 10123 TORINO (Italy)
tel: +39 011 6704542
http://penelope.unito.it/friard/
http://orcid.org/0000-0002-0374-9872

**Comment by bfranks80** [comment]
Thank you very much for the quick reply!  We are mainly using the current
version (v7.10.2), but some students can only get older versions to open,
eg v5.0.1.

Thanks again for your attention.

**Comment by olivierfriard** [comment]
Hello,

5.0.1 is very old now and my advice is to not use it.
You should have to accept the use of the program twice or more in
the "security & privacy" settings because BORIS embed ffmpeg that requires
authorization too.

o
-- 
University of Torino
Life Sciences and Systems Biology department
Via dell'Accademia Albertina, 13 10123 TORINO (Italy)
tel: +39 011 6704542
http://penelope.unito.it/friard/
http://orcid.org/0000-0002-0374-9872

**Comment by bfranks80** [comment]
Ok, good to know.  Thank you very much for your reply and advice.

All the best,

**Comment by olivierfriard** [comment]
hi,
the last versions of BORIS run on a Raspberry Pi computer (a low-cost computer that plugs into a TV or a computer monitor).
See http://www.boris.unito.it/pages/raspberry_pi_4.html for details
